.changelog/unreleased/bug-fixes/indexer-respect-height-params-on-query.md

```diff
@@ -0,0 +1,3 @@
+### BUG FIXES
+
+- `[state/indexer]` Respect both height params while quering for events
```

Comment:
Please ensure there's a link to this PR at the end of the changelog entry line, e.g. https://github.com/cometbft/cometbft/blob/58cfaf973bcd88808e130ea454fe7fc5015e3078/.changelog/unreleased/enhancements/1096-state-background-pruning.md